blog/02-16-2023-Exploring-Celo-in-the-Gaming-Industry-for-In-Game-Assets-and-P2P-Trading/Exploring-Celo-in-the-Gaming-Industry-for-In-Game-Assets-and-P2P-Trading.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Exploring Celo in the Gaming Industry for In-Game Assets and P2P Trading
+description: Learn how Celo is utilized in the Gaming Industry for In-Game Assets and P2P Trading
+authors:
+- name: Oluwalana Ajayi
+  title: Web3 Content Writer
+  url: https://github.com/lanacreates
+  image_url: https://github.com/lanacreates.png
+tags: [celosage, celowallet, celo, cusd, ledger]
+hide_table_of_contents: false
+slug: /tutorials/Exploring-Celo-in-the-Gaming-Industry-for-In-Game-Assets-and-P2P-Trading
```

Comment:
/tutorials/exploring-celo-in-the-gaming-industry-for-In-game-assets-and-p2p-trading

It has to be in small case, change this